package.json

```diff
@@ -18,7 +18,7 @@
     "install-python-deps": "lerna run install-python-deps --concurrency 1",
     "lint": "npm run lint-package-json && npm run eslint && lerna run python-lint && npm run lint-md",
     "lint-md": "markdownlint docs/**/*.md docs/*.md tf-modules/**/*.md tf-modules/*.md",
-    "lint-package-json": "npmPkgJsonLint .",
+    "lint-package-json": "true",
```

Comment:
This is open for discussion - adding the async operations package results in a lerna link chain that exceeds a default search built into this module (see:  https://github.com/tclindner/npm-package-json-lint/issues/342 / 9e3c42bdb768d12ca60a3030747afb685fdca453 ) for more

Reply:
I discussed this with Menno, and based on that conversation feel comfortable leaving this with a TD ticket in Jira to address.    If there's objections, we can have that conversation here/elsewhere.

Reply:
It looks like this supports globbing, so could it change to this instead?

`npmPkgJsonLint packages/* tasks/* example lambdas/*`

I wouldn't **think** that would recurse into packages/api/ecs/async-operation

Reply:
Sadly it appears to, I'm 99% sure I tested that exact approach (among others), but I'll give it another whirl to be sure once I've sorted the cyclic issue.

Reply:
*Apparently* some of the local build changes have 'sorted' the problem.   We'll see if it's reproducible in CI:

e9c023239